charts/pulsar/templates/zookeeper/zookeeper-configmap.yaml

```diff
@@ -45,6 +45,7 @@ data:
   {{- if .Values.zookeeper.reconfig.enabled }}
   PULSAR_PREFIX_reconfigEnabled: "true"
   PULSAR_PREFIX_quorumListenOnAllIPs: "true"
+  PULSAR_PREFIX_skipACL: "yes"
```

Comment:
Do you need to consider adding an option to control whether it is turned on or off, and is there a security risk in setting it to skipACL by default?

Reply:
if we need to enable dynamic reconfig then we need to either set skipACL or explicitly set a password, I think the risk is pretty much the same, and zk is not exposed to public network though